src/helpers/utils.js

```diff
@@ -25,13 +25,14 @@ exports.getLocaleCodes = getLocaleCodes
 
 /**
  * Retrieve page's options from the module's configuration for a given route
- * @param  {Object} route    Route
- * @param  {Object} pages    Pages options from module's configuration
- * @param  {Array} locales   Locale from module's configuration
- * @param  {String} pagesDir Pages dir from Nuxt's configuration
- * @return {Object}          Page options
+ * @param  {Object} route         Route
+ * @param  {Object} pages         Pages options from module's configuration
+ * @param  {Array} locales        Locale from module's configuration
+ * @param  {String} pagesDir      Pages dir from Nuxt's configuration
+ * @param  {String} defaultLocale Pages dir from Nuxt's configuration
```

Comment:
the description seems to have been copy-pasted from the previous line 😕

Reply:
fixed :stuck_out_tongue_winking_eye: